src/rgw/rgw_amqp.cc

```diff
@@ -322,6 +322,10 @@ std::string to_string(amqp_status_enum s) {
       return "AMQP_STATUS_SSL_CONNECTION_FAILED";
     case _AMQP_STATUS_SSL_NEXT_VALUE:
       return "AMQP_STATUS_INTERNAL"; 
+#if AMQP_VERSION_MAJOR >= 0 && AMQP_VERSION_MINOR >= 11
```

Comment:
please write the condition similarly to line 303

Reply:
Thank you for the given commit, the line has been modified accordingly.